python/orca/example/learn/pytorch/fashion_mnist/fashion_mnist.py

```diff
@@ -38,26 +38,26 @@
 from bigdl.orca.learn.trigger import EveryEpoch
 
 
-def train_data_creator(config, batch_size):
+def train_data_creator(config, batch_size, args):
     transform = transforms.Compose(
         [transforms.ToTensor(),
          transforms.Normalize((0.5,), (0.5,))])
 
-    trainset = torchvision.datasets.FashionMNIST('./data',
-                                                 download=True,
+    trainset = torchvision.datasets.FashionMNIST(args.data_dir,
+                                                 download=args.download,
                                                  train=True,
                                                  transform=transform)
     trainloader = torch.utils.data.DataLoader(trainset, batch_size=batch_size,
                                               shuffle=True, num_workers=2)
     return trainloader
 
 
-def validation_data_creator(config, batch_size):
+def validation_data_creator(config, batch_size, args):
```

Comment:
`def validation_data_creator(config, batch_size, download)` is better

Reply:
Updated in new commit.